src/components/TodoList/TodoList.tsx

```diff
@@ -16,85 +28,58 @@ export const TodoList: React.FC = () => (
-            <span className="icon">
-              <i className="far fa-eye-slash" />
-            </span>
-          </button>
-        </td>
-      </tr>
-
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">1</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">delectus aut autem</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
+      {todos.map(({
+        id,
+        title,
+        completed,
+        userId
+      }) => (
+        <tr
+          data-cy="todo"
+          key={id}
+          className={classNames({
+            'has-background-info-light': id === selectedTodoId,
+          })}
+        >
+          <td className="is-vcentered">{id}</td>
+          <td className="is-vcentered">
             <span className="icon">
-              <i className="far fa-eye" />
+              <i className={classNames({
+                'fas fa-check': completed,
+              })}
+              />
             </span>
-          </button>
-        </td>
-      </tr>
-
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">6</td>
-        <td className="is-vcentered" />
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-danger">
-            qui ullam ratione quibusdam voluptatem quia omnis
-          </p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
-
-      <tr data-cy="todo" className="">
-        <td className="is-vcentered">8</td>
-        <td className="is-vcentered">
-          <span className="icon" data-cy="iconCompleted">
-            <i className="fas fa-check" />
-          </span>
-        </td>
-        <td className="is-vcentered is-expanded">
-          <p className="has-text-success">quo adipisci enim quam ut ab</p>
-        </td>
-        <td className="has-text-right is-vcentered">
-          <button data-cy="selectButton" className="button" type="button">
-            <span className="icon">
-              <i className="far fa-eye" />
-            </span>
-          </button>
-        </td>
-      </tr>
+          </td>
+          <td className="is-vcentered is-expanded">
+            <p className={classNames({
+              'has-text-success': completed,
+              'has-text-danger': !completed,
+            })}
+            >
+              {title}
+            </p>
+          </td>
+          <td className="has-text-right is-vcentered">
+            <button
+              data-cy="selectButton"
+              className="button"
+              type="button"
+              onClick={() => onSelectTodo({
```

Comment:
just take a whole todo
```suggestion
              onClick={() => onSelectTodo(todo)
```